scrapers/AussieFelatioQueens.yml

```diff
@@ -0,0 +1,24 @@
+name: Aussie Fellatio Queens
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - https://aussiefellatioqueens.com/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //h2[@class='video-title fleft text-color-3']/span[2]/text()
+      Performers:
+        Name: 
+          selector: //span/span/a/span/text()
+      Details:
+        selector: //h3[@class='text-video-desc text-color-5 full-desc']/span/text()
```

Comment:
This selector didn't work on certain scenes like `https://aussiefellatioqueens.com/scene/8074313/swallow-booth-rubi-valentine-1st-visit-cumshot-compilation`

This seems to solve that issue:
```
selector: //h3[contains(@class,'text-video-desc') and not(contains(@class,'short-desc'))]/span/text()
```